Main-rpn/Main-rpn/Main-rpn.cpp

```diff
@@ -0,0 +1,97 @@
+#include <iostream>
+#include <fstream>
+
+#include <math.h>
+#include <string>
+#include <stack>
+
+int calculate(const std::string &infix)
+{
+    std::cout << infix << "\n";
+    std::string postfix = "";
+
+    std::stack<char> operators;
+
+    // Convert from infix to postfix notation
+    for (int i = 0; i < infix.size(); i++)
+    {
+        if (infix[i] >= 48 && infix[i] <= 57)
+        {
+            postfix += infix[i];
+            postfix += ' ';
+        }
+        else if (infix[i] == '+' || infix[i] == '-' || infix[i] == '*' || infix[i] == '/' || infix[i] == '^')
+            operators.push(infix[i]);
+        if (infix[i] == ')' || i == infix.size() - 1)
```

Comment:
how can we deal with unbalanced parenthesis? e.g. (((9 - 4) ^ 3) - 2 * 4 or ((9 - 4) ^ 3) - 2) * 4